src/config.c

```diff
@@ -26,6 +26,65 @@
 #define RS_MAX_CONFIG_TRIGGERS 1 // Increase this if you need more triggers
 RSConfigExternalTrigger RSGlobalConfigTriggers[RS_MAX_CONFIG_TRIGGERS];
 
+typedef struct {
+  const char *FTConfigName;
+  const char *ConfigName;
+} configPair_t;
+
+configPair_t __configPairs[] = {
```

Comment:
On next PR in this area, we can wrap the registration functions (for Bool, Enum, Numeric and String) with a wrapper that will also receive the "old" name and add to an array of __configPairs - this way all would be defined in one place.